### ISSUE TITLE
Ask for some implementation details

### ISSUE THREAD

**Issue by polaris-973** [omarfoq/knn-per#9]
Hi! Thanks for your work first!

I would like to confirm a few implementation details with you:

1. Is it the full dataset of client or just the training set that is used in creating the datastore, as I see on the paper that it is the local full dataset and the code implementation is based on the training set to extract the data features, if I am not mistaken.
2. I would like to confirm with you the workflow of the _kNN-Per_ method: first train the global model  (are classifier included here?) based on the traditional FL protocol. When performing classification inference, the test set data is also converted into feature vectors and then correspoding kNNs are queried in the data store, and the final classification is performed according to the voting principal (suppose the global model prediction interpolation is ignored). I hope you kindly point out if I have misunderstood~

Looking forward to your reply soon, thanks again!
